inductiva/tasks/task.py

```diff
@@ -379,18 +381,29 @@ def download_outputs(
             rm_downloaded_zip_archive: Whether to remove the archive after
             uncompressing it. If uncompress is False, this argument is ignored.
         """
-        api_response = self._api.download_task_output(
-            path_params=self._get_path_params(),
-            query_params={
-                "filename": filenames or [],
-            },
-            stream=True,
-            skip_deserialization=True,
-        )
-        # use raw urllib3 response instead of the generated client response, to
-        # implement our own download logic (with progress bar, first checking
-        # the size of the file, etc.)
-        response = api_response.response
+        api_response = self._api.get_output_download_url(
+            path_params=self._get_path_params(),)
+
+        download_url = api_response.body.get("url")
+
+        if download_url is None:
+            raise RuntimeError(
+                "The API did not return a download URL for the task outputs.")
+
+        logging.debug("\nDownload URL: %s\n", download_url)
+
+        try:
```

Comment:
suggestion: create small function to detect if file_server_is available

Reply:
We moved this logic to the backend since it is also needed for uploads.

https://github.com/inductiva/inductiva-web-api/pull/852